tests/Unit/Helpers/Elliptic/FirstOrderSystem.hpp

```diff
@@ -0,0 +1,279 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Helper functions to test elliptic first-order systems
+
+#pragma once
+
+#include "Framework/TestingFramework.hpp"
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "Elliptic/FirstOrderComputeTags.hpp"
+#include "Elliptic/FirstOrderOperator.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace TestHelpers::elliptic {
+
+template <typename System>
+void test_first_order_fluxes_computer(
```

Comment:
Should these and the `TestHelpers::elliptic` namespace be documented? `TestHelpers::elliptic` can go in the `TestingFrameworkGroup` and then the functions can just be in this namespace (no group). It would be good to describe what the test does in the documentation. As is I need to check the C++ and figure out what the test does, then determine if that's what you want it to do :P